marketplace-service/src/main/java/com/axonivy/market/controller/FeedbackController.java

```diff
@@ -72,25 +90,41 @@ public ResponseEntity<PagedModel<FeedbackModel>> findFeedbacks(@PathVariable(ID)
   }
 
   @GetMapping(BY_ID)
-  public ResponseEntity<FeedbackModel> findFeedback(@PathVariable(ID) String id) {
+  @Operation(summary = "Find all feedbacks by product id", description = "Get all feedbacks by product id(from meta.json) which is used in mobile viewport.")
+  public ResponseEntity<FeedbackModel> findFeedback(@PathVariable(ID)
+                                                    @Parameter(description = "Product id (from meta.json)", example = "portal", in = ParameterIn.PATH)
+                                                    String id) {
     Feedback feedback = feedbackService.findFeedback(id);
     return ResponseEntity.ok(feedbackModelAssembler.toModel(feedback));
   }
 
-  @Operation(summary = "Find all feedbacks by user id and product id")
   @GetMapping()
-  public ResponseEntity<FeedbackModel> findFeedbackByUserIdAndProductId(@RequestParam(USER_ID) String userId,
-      @RequestParam("productId") String productId) {
+  @Operation(summary = "Find all feedbacks by user id and product id", description = "Get current user feedback on target product.")
+  public ResponseEntity<FeedbackModel> findFeedbackByUserIdAndProductId(@RequestParam(USER_ID)
+                                                                        @Parameter(description = "Id of current user from DB", example = "1234", in = ParameterIn.QUERY)
+                                                                        String userId,
+                                                                        @RequestParam("productId")
+                                                                        @Parameter(description = "Product id (from meta.json)", example = "portal", in = ParameterIn.QUERY)
+                                                                        String productId) {
     Feedback feedback = feedbackService.findFeedbackByUserIdAndProductId(userId, productId);
     return ResponseEntity.ok(feedbackModelAssembler.toModel(feedback));
   }
 
   @PostMapping
-  public ResponseEntity<Void> createFeedback(@RequestBody @Valid FeedbackModel feedback,
-      @RequestHeader(value = AUTHORIZATION) String authorizationHeader) {
+  @Operation(summary = "Create user feedback", description = "Save user feedback of product with their token from Github account.")
+  @io.swagger.v3.oas.annotations.parameters.RequestBody(description = "Example request body for feedback", content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE, schema = @Schema(implementation = FeedbackModel.class)))
+  @ApiResponses(value = {
+          @ApiResponse(responseCode = "201", description = "Successfully created user feedback"),
+          @ApiResponse(responseCode = "401", description = "Unauthorized request")})
+  public ResponseEntity<Void> createFeedback(@RequestBody
```

Comment:
```
  @Getter
  @Setter
  @Schema(name = "CustomFeedbackModel", description = "Custom model for user feedback")
  public static class CustomFeedbackModel {

    @Schema(description = "ID of the product", example = "adobe-acrobat-connector")
    private String productId;

    @Schema(description = "Content of the feedback", maxLength = 500, example = "This is a great product!")
    private String content;

    @Schema(description = "Rating given by the user", minimum = "1", maximum = "5", example = "4")
    private int rating;
  }
```
  
  Could you please add this model for me to CustomFeedbackModel, we just need 3 property productId, content and rating for it